mysite/profile/models.py

```diff
@@ -647,10 +648,7 @@ def get_url_or_guess(self):
 
     def save_and_check_for_duplicates(self):
         # FIXME: Cache summaries in the DB so this query is faster.
-        duplicates = [citation for citation in
-                      Citation.objects.filter(
-                          portfolio_entry=self.portfolio_entry)
-                      if (citation.pk != self.pk) and (citation.summary == self.summary)]
+        duplicates = [citation for citation in Citation.objects.filter(portfolio_entry=self.portfolio_entry) if (citation.pk != self.pk) and (citation.summary == self.summary)]
```

Comment:
I definitely preferred the way this one was before. This line is so long I have to scroll!